src/applications/static-pages/download-1095b/components/App/index.unit.spec.js

```diff
@@ -22,7 +22,7 @@ describe('App component', () => {
       },
       profile: {
         loa: {
-          current: 1,
+          current: null,
         },
```

Comment:
Unauthed mock users don't need an LOA value. This change fixes a unit test tat breaks in CI.